viewer/viewer.js

```diff
@@ -114,6 +114,10 @@ if (internals.elasticBase[0].lastIndexOf('http', 0) !== 0) {
   internals.elasticBase[0] = "http://" + internals.elasticBase[0];
 }
 
+function isProduction() {
+  return app.get('env') === 'production';
```

Comment:
shouldn't this be NODE_ENV instead of env?

Reply:
oh does express change it from NODE_ENV to just env?

Reply:
According to: https://expressjs.com/en/guide/migrating-4.html

```The app.configure() function has been removed. Use the process.env.NODE_ENV or app.get('env') function to detect the environment and configure the app accordingly.```